### PR TITLE
Fix Dependabot

### PR DESCRIPTION
**Pull request by lucacome**
- Adds configuration for all the folders
- Moves versions to the `FROM` line in Dockerfiles so dependabot can keep them up to date.

### CODE REVIEW

File: semantic-conventions/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,4 @@
-ARG ALPINE_VERSION=3.12
-ARG PYTHON_VERSION=3.8.5
-
-FROM python:${PYTHON_VERSION}-alpine${ALPINE_VERSION}
```
Comment (Oberon00) [inline semantic-conventions/Dockerfile:4]: Why are you removing the ARGs? Is dependabot not able to process them?

**Comment by lucacome** [comment]
can we merge it @Oberon00 ? 🙂

**Comment by lucacome** [comment]
friendly ping 🙂

**Comment by Oberon00** [comment]
Hi, I'm not really the right reviewer for the Docker stuff. Maybe @bogdandrutu @tigrannajaryan?

**Comment by lucacome** [comment]
@bogdandrutu @tigrannajaryan any way we can get this merged?

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**